app/utils/strings.ts

```diff
@@ -4,3 +4,15 @@
 export function nonBreakingString(s: string) {
     return s.replace(' ', '\xa0');
 }
+
+export function generateRandomString(length: number) {
+    let result = '';
+    let counter = 0;
+    const chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+    const charsLenght = chars.length;
+    while (counter < length) {
+        result += chars.charAt(Math.floor(Math.random() * charsLenght));
```

Comment:
This needs to use a CSPRNG. `Math.random` is notoriously predictable.

Reply:
Can you double check this lib? https://www.npmjs.com/package/react-native-securerandom